Testscripts/Windows/PERF-NETWORK-THROUGHPUT-MULTICONNECTION-NTTTCP.ps1

```diff
@@ -60,6 +60,20 @@ function Main {
         Provision-VMsForLisa -allVMData $allVMData -installPackagesOnRoleNames "none"
         #endregion
 
+        # Set systemd logind.conf
+        $setSystemdConfig = "sed -i '`$aUserTasksMax=122880' /etc/systemd/logind.conf"
+        foreach ($vmData in $allVMData) {
+            Run-LinuxCmd -ip $vmData.PublicIP -port $vmData.SSHPort -username "root" `
+                    -password $password -command $setSystemdConfig | Out-Null
+        }
+
+        # Restart VM to apply systemd setting
+        if (-not $TestProvider.RestartAllDeployments($allVMData)) {
+            Write-LogErr "Unable to connect to VM after restart!"
+            $currentTestResult.TestResult = Get-FinalResultHeader -resultarr "ABORTED"
```

Comment:
$currentTestResult.TestResult = "ABORTED" is enough

Reply:
Fixed in 60aa439c0f5a03dd8441bb3f1d990eb6b668b934